src/TwitterStream.js

```diff
@@ -18,13 +18,14 @@ class DeferredPromise {
 }
 
 class TwitterStream {
-  constructor(connect, close) {
+  constructor(connect, close, wait) {
```

Comment:
Let's make this an options parameter as we may want to support more options in the future.

```js
constructor(connect, close, options) {
  const { timeout = 20000 } = options;

  ...
}
```